lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.ts

```diff
@@ -234,8 +228,7 @@ export class SearchDateRangeComponent implements SearchWidget, OnInit, OnDestroy
     }
 
     setLocale(locale) {
```

Comment:
please provide the missing typings here, probably "locale: string"

Reply:
added type here